### DISCUSSION TITLE
Ping Pong

### DISCUSSION THREAD

**Discussion by palo999** [arthurkushman/php-wss#85] [Q&A] [answered]
Please, how could I maintain a connection using ping-pong? I can't find the interval, and I don't know where or what to write. 
Thank you for your response.

**Comment by arthurkushman** [comment, marked as answer]
If u r trying to set up a server timeout - just use these options (u can find them in readme file):
```php
$config = new ServerConfig();
$config->setClientsPerFork(2500);
$config->setStreamSelectTimeout(2 * 3600);
```

if otherwise u need to set up timeout on client:
```php
$config = new ClientConfig();
$config->setFragmentSize(8096);
$config->setTimeout(15);
```
